qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/parameter/scanner/ParameterScanner.java

```diff
@@ -15,27 +15,40 @@
 
 import com.redhat.qute.parser.scanner.AbstractScanner;
 
+/**
+ * Parameter scanner.
+ * 
+ * @author Angelo ZERR
+ *
+ */
 public class ParameterScanner extends AbstractScanner<TokenType, ScannerState> {
 
 	private static final Predicate<Integer> PARAMETER_NAME_OR_VALUE_PREDICATE = ch -> {
 		return ch != ' ' && ch != '=';
 	};
-	
-	public static ParameterScanner createScanner(String input) {
-		return createScanner(input, 0, input.length());

```

Comment:
or you could keep that method and call `createScanner(input, 0, input.length(), false);`, thus keeping ParameterScannerTest.java simple

Reply:
fixed